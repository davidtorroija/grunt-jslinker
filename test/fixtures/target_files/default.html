<!doctype html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Document</title>
</head>
<body>

    <!--MYSCRIPTS-->
    <script src='test/fixtures/js/123.js'></script>
    <script src='test/fixtures/js/excluded/test_exclude.js'></script>
    <script src='test/fixtures/js/test.js'></script>
    <script src='test/fixtures/js/testing.js'></script>
    <!--MYSCRIPTS END-->

</body>
</html>
