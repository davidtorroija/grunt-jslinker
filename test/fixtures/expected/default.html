<!doctype html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Document</title>
</head>
<body>

    <!--MYSCRIPTS-->
    <script src='fixtures/js/123.js'></script>
    <script src='fixtures/js/excluded/test_exclude.js'></script>
    <script src='fixtures/js/test.js'></script>
    <script src='fixtures/js/testing.js'></script>
    <!--MYSCRIPTS END-->

</body>
</html>
